gcalcli/__init__.py

```diff
@@ -1,5 +1,3 @@
 __program__ = 'gcalcli'
 __version__ = 'v4.3.0'
 __author__ = 'Eric Davis, Brian Hartvigsen, Joshua Crowgey'
-__API_CLIENT_ID__ = '232867676714.apps.googleusercontent.com'
```

Comment:
@insanum could you confirm this client ID is completely irrelevant nowadays? If we did want to set up a centralized project, would we be better off creating a new one instead of trying to revive this?